codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/Ec2Query.kt

```diff
@@ -80,7 +82,7 @@ class Ec2QueryProtocol(private val protocolConfig: ProtocolConfig) : Protocol {
         }
 
     override fun parseEventStreamGenericError(operationShape: OperationShape): RuntimeType =
-        RuntimeType.forInlineFun("parse_event_stream_generic_error", "xml_deser") { writer ->
+        RuntimeType.forInlineFun("parse_event_stream_generic_error", xmlDeserModule) { writer ->
```

Comment:
could add a method overload or have an optional `publicity` param as the next option if you want to remove the boilerplate.

Or `RuntimeType.publicInlineFunction` etc.